components/massmint/uploadCompressedMedia/UploadCompressedMedia.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div>
+    <NeoCollapse :disabled="disabled">
+      <div class="is-flex">
+        {{ $t('massmint.uploadPics') }}
+        <NeoIcon
+          v-if="showCheckmark"
+          icon="circle-check"
+          size="small"
+          variant="success"
+          class="ml-3" />
+        <div v-else class="icon-placeholder ml-3" />
+      </div>
+      <template #content>
+        <DragDrop
+          accept=".zip"
+          :loading="loading"
+          @fileSelected="onFileSelected">
+          <template #title>
+            <span class="mb-4 has-text-left">{{
+              $t('massmint.uploadzipTip')
+            }}</span>
+            <span class="has-text-left"
+              ><b>{{ $t('massmint.supportedFormats') }}</b>
+              {{ acceptedMediaFormatsString }}</span
+            >
+          </template>
+        </DragDrop>
+      </template>
+    </NeoCollapse>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { NeoCollapse, NeoIcon } from '@kodadot1/brick'
+import { useZipFileValidator } from './useZipValidator'
+import { notificationTypes, showNotification } from '@/utils/notification'
+import DragDrop from '@/components/shared/DragDrop.vue'
+
+withDefaults(
+  defineProps<{
+    disabled?: boolean
+  }>(),
+  {
+    disabled: false,
+  }
+)
+
+const loading = ref(false)
+const showCheckmark = ref(false)
+
+const acceptedMediaFormatsString =
+  'BMP, GIF, JPEG, PNG, SVG, TIFF, WEBP, MP4, OGV, QUICKTIME, WEBM, GLB, FLAC, MP3, JSON'
```

Comment:
This string could be generated auto from `validFormats` list at `components/massmint/uploadCompressedMedia/useZipValidator.ts`

Reply:
True that

Reply:
done